packages/@aws-cdk/aws-cloudwatch/lib/alarm.ts

```diff
@@ -51,18 +51,24 @@ export enum ComparisonOperator {
   /**
    * Specified statistic is lower than or greater than the anomaly model band.
    * Used only for alarms based on anomaly detection models
+   *
+   * @deprecated Use AnomalyDetectionAlarm instead.
```

Comment:
I don't see a reason to do this. Can you explain why you want to deprecate these when they're used more widely than just in this context?